src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -240,8 +241,8 @@ public String remoteMutePlayer(final String playerName, final int minutes, final
     }
     final String localPassword = System.getProperty(GameRunner.LOBBY_GAME_SUPPORT_PASSWORD, "");
     final String encryptedPassword = MD5Crypt.crypt(localPassword, salt);
-    // milliseconds (48 hours max)
-    final long expire = System.currentTimeMillis() + (Math.max(0, Math.min(60 * 24 * 2, minutes)) * 1000 * 60);
+    // (48 hours max)
+    Instant expireInstant = Instant.now().plus(Duration.ofMinutes(Math.min(60 * 24 * 2, minutes)));
```

Comment:
Can we rename this back to `expire`?  It looks like the same name is being used below.